querybook/webapp/ui/Comment/Comment.tsx

```diff
@@ -68,6 +77,14 @@ export const Comment: React.FunctionComponent<IProps> = ({
         [reactions]
     );
 
+    const handleUndoDeleteComment = React.useCallback(() => {
+        if (recentlyArchived) {
```

Comment:
nit: probably not needed to check this again since this function is only available when recentlyArchived is true